neural_modelling/src/neuron/population_table/population_table_binary_search_impl.c

```diff
@@ -24,9 +24,12 @@ typedef struct master_population_table_entry {
     uint32_t mask;
     uint16_t start;
     uint16_t count;
+    uint16_t core_mask;
+    uint16_t n_neurons_and_mask_shift;
 } master_population_table_entry;
 
 typedef uint32_t address_and_row_length;
+#define INVALID_ADDRESS_AND_ROW_LENGTH 0xFFFFFFFF
```

Comment:
THis magic flag is used in the generator_data for offsets. but not referenced. bad bad bad

Reply:
This is a different 0xFFFFFFFF from the one there.